tests/browser/http/test_content_disposition.py

```diff
@@ -122,17 +126,17 @@ def test_inlwithasciifilenamepdf(self):
         with the latest Acrobat Reader plugin, or, in the case of Chrome, using
         the built-in PDF handler).
         """
-        self._check_filename('inline; filename="foo.pdf"', "foo.pdf")
-
+        header_checker.check_filename('inline; filename="foo.pdf"', "foo.pdf",
+                                      expected_inline=True)
 
-class AttachmentTests(AttachmentTestCase):
 
+class TestAttachment:
```

Comment:
Yep, in fact when I started converting a few classes I subclassed them from `object` out of habit, then I realized we are using py3k and don't need that anymore (in fact I think it is no longer considered good style). :smile: